login/login.js

```diff
@@ -0,0 +1,129 @@
+let mysql = require('mysql');
+
+let connectionConfig = require('./ConfigConnection.json');
+
+const connection = mysql.createConnection(connectionConfig);
+const LoginRequest  = require("./LoginRequest.js");
+const FrontQuery = require("./FrontQuery.js");
+const UserData = require("./UserData.js");
+
+
+const Sequelize = require("sequelize");
+const sequelize = new Sequelize("web","root","Edding16", {
+    dialect: "mysql",
+    host: "localhost",
+    logging: false,
+    define: {
+        timestamps: false,
+    }
+});
+
+
+
+
+    const Roles = sequelize.define("roles", {
+        id:{type: Sequelize.INTEGER,
+            primaryKey: true,
+            autoIncrement: true,
+            allowNull: false
+        },
+        role_name:{
+            type: Sequelize.STRING,
+            allowNull: false
+        }
+    });
+
+    const User = sequelize.define("users", {
+    id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true,
+        allowNull: false
+    },
+    first_name: {
+        type: Sequelize.STRING,
+        allowNull: false
+    },
+    second_name:{
+        type: Sequelize.STRING,
+        allowNull: false,
+
+    },
+    role_id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        references:{
+            model: 'roles',
+            key: 'id'
+        }
+    },
+    login:{
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique:true
+    },
+    password:{
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique:true
+    },
+    birthday: {
+        type: Sequelize.DATE,
+        allowNull: false
+    }
+});
+
+    sequelize.sync().then(result=>{
+        console.log(result);
+    })
+        .catch(err=> console.log(err));
+
+
+
+
+/*function addUser(userData,User) {
+
+    User.create({
+        first_name: userData.name,
+        second_name:userData.surname,
+        birthday: userData.birthday,
+        login:userData.login,
+        password: userData.password,
+        role_id:userData.roleId
+    }).then(res=>{
+        console.log(res);
+    }).catch(err=>console.log(err));
+}
+
+let userData = new UserData(1,"Bob2123","Bob123","Jon","Travolta",1999-1-12);
+
+addUser(userData,User);
+*/
+
+
+let loginRequest = new  LoginRequest('Bob123','Bob123');
+
+
+function SingIn (loginRequest)
+{
+    User.findOne({raw:true,where: {login: loginRequest.login,password:loginRequest.password}})
+        .then(User=>{
+            if(!User){
+                console.log("Данные введены неправильно");
```

Comment:
nit: I would write all of the strings in english to be consistent and don't worry about unicode.

Reply:
it product for RUSSIAN people.our developers,tester,and final user-russians